rxnetty/src/main/java/io/reactivex/netty/client/ConnectionPoolImpl.java

```diff
@@ -110,54 +120,73 @@ public ConnectionPoolImpl(RxClient.ServerInfo serverInfo, PoolConfig poolConfig,
     @Override
     public Observable<ObservableConnection<I, O>> acquire() {
 
-        if (isShutdown.get()) {
+        if (isShutdownRequested.get()) {
```

Comment:
It is already in there but I wanted to fail fast. ```if (!lockRetrieved || isShutdownRequested.get())``` Do you think it is enough in the call method?